server/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,18 @@
+name: "OpenAPI Generator - REST Server Generator"
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+metadata:
+  keywords:
+    - "openapi"
+    - "openapi-generator-server"
+    - "rest"
+    - "rest-client"
```

Comment:
I think we can remove the `rest-client` and to add `server` or something similar (indicating a server).